.drone.yml

```diff
@@ -42,43 +43,38 @@ pipeline:
 
   # pr
 
-  sbt-update:
-    <<: *sbtenv
-    commands:
-      - sbt update
-
   compile:
     <<: *sbtenv
     commands:
-      - sbt rholang/bnfc:generate compile test:compile it:compile doc
-
-  run-unit-tests:
-    <<: *sbtenv
-    commands:
-      - sbt test it:test
-
-  run-code-analysis:
-    <<: *sbtenv
-    commands:
-      - sbt scalafmtCheck coverage
-      - ./scripts/ci/codecov-upload
+      - sbt update rholang/bnfc:generate compile
 
   # push
 
-  package:
+  docker-build:
     <<: *sbtenv
     commands:
-      - sbt node/docker:publishLocal node/debian:packageBin node/universal:packageZipTarball
-      - mkdir -p artifacts
-      - cp node/target/rnode_*_all.deb node/target/universal/*.tgz artifacts/
+      # Skip the `doc` task - it's being invoked explicitly later
+      - SKIP_DOC=true sbt node/docker:publishLocal
     volumes:
       - /var/cache/rchain-build/.sbt:/root/.sbt
       - /var/cache/rchain-build/.ivy2:/root/.ivy2
       - /var/run/docker.sock:/var/run/docker.sock
     when:
       event: [ push, tag ]
 
+  # pr
+
+  run-unit-tests:
+    group: parallel
```

Comment:
How does this parallelize the tests?

Reply:
https://0-8-0.docs.drone.io/pipelines, see Parallel Execution